feedback/lib/src/feedback_widget.dart

```diff
@@ -117,117 +132,153 @@ class FeedbackWidgetState extends State<FeedbackWidget>
-                  colors: widget.drawColors,
-                  onColorChanged: (color) {
-                    setState(() {
-                      painterController.drawColor = color;
-                    });
-                    _hideKeyboard(context);
-                  },
-                  onUndo: () {
-                    painterController.undo();
-                    _hideKeyboard(context);
-                  },
-                  onClearDrawing: () {
-                    painterController.clear();
-                    _hideKeyboard(context);
-                  },
-                  onControlModeChanged: (mode) {
-                    setState(() {
-                      this.mode = mode;
-                      _hideKeyboard(context);
-                    });
-                  },
-                  onCloseFeedback: () {
-                    _hideKeyboard(context);
-                    BetterFeedback.of(context).hide();
-                  },
-                ),
-              ),
-              // only display if feedback is visible or this widget is still
-              // animating out
-              if (widget.isFeedbackVisible || !animation.isDismissed)
-                Positioned(
-                  key: const Key('feedback_bottom_sheet'),
-                  left: 0,
-                  // Make sure the input field is always visible,
-                  // especially if the keyboard is shown
-                  bottom: MediaQuery.of(context).viewInsets.bottom,
-                  right: 0,
-                  height: MediaQuery.of(context).size.height *
-                      // height should be screen size minus the bottom edge of
-                      // screenshot widget:
-                      //   1 - (scaleOrigin + height*scaleFactor)
-                      (1 - (.35 / 2 + 1.65 / 2 * .65)),
-                  child: SlideTransition(
-                    position: Tween(begin: const Offset(0, 1), end: Offset.zero)
-                        .animate(animation),
-                    child: Padding(
-                      padding: const EdgeInsets.only(top: 24),
-                      child: FeedbackBottomSheet(
-                        feedbackBuilder: widget.feedbackBuilder,
-                        onSubmit: (
-                          String feedback, {
-                          Map<String, dynamic>? extras,
-                        }) async {
-                          await _sendFeedback(
-                            context,
-                            BetterFeedback.of(context).onFeedback!,
-                            screenshotController,
-                            feedback,
-                            widget.pixelRatio,
-                            extras: extras,
-                          );
-                          painterController.clear();
-                        },
+                builder: (context, screenshotChild) {
+                  return CustomMultiChildLayout(
```

Comment:
This section has gotten huge-it'd probably be a good candidate for breaking out into a separate widget. I could add that to this PR or we could put it off to a future PR.

Reply:
Your call